app.py

```diff
@@ -161,7 +168,7 @@ def get_current_frame_range(self, engine):
 
         return (current_in, current_out)
 
-    def set_frame_range(self, engine, in_frame, out_frame):
+    def set_frame_range(self, engine, in_frame, out_frame, prev_in, prev_out):
```

Comment:
Yes. If we want to run an app, we normally need to use `app.run_app`. And normally, we "should" not use other apps methods (Note the "normally")